cli.ts

```diff
@@ -559,24 +656,22 @@ export const run = (env: EnvVars, inputArgs: DenoArgs, i18n: i18n) => {
             (cliConfig: CLIConfig) => pipe(
                 cliConfig,
                 parseArgs,
-                map (sanitizeArgs),
+                chain(SanitizedArgs.of),
                 chain (initArgs => {
                     if (initArgs.debug) debugMode(true)
 
                     if (initArgs.version) {
                         log(initArgs.setVersion)
-                        return resolve(initArgs)
+                        return resolve(initArgs) as Future<TaqError.t, SanitizedArgs.t>
```

Comment:
Let's create an abstraction function in taqueria utils that calls resolve and sets the type of the rejection path to taq error.

Reply:
Done!